src/graph/query_graph.c

```diff
@@ -376,6 +376,39 @@ uint QueryGraph_EdgeCount(const QueryGraph *qg) {
 	return array_len(qg->edges);
 }
 
+GrB_Matrix QueryGraph_MatrixRepresentation(const QueryGraph *qg) {
+	assert(qg);
+
+	// Make a clone of the given graph as we're about to modify it.
```

Comment:
can you point where is the modification of the query graph in this function?

Reply:
`for(uint i = 0; i < node_count; i++) qg_clone->nodes[i]->labelID = i;`